fission/prettier.config.js

```diff
@@ -9,7 +9,15 @@ const config = {
     bracketSpacing: true,
     bracketSameLine: false,
     arrowParens: "avoid",
-    printWidth: 120
+    printWidth: 120,
+    overrides: [
+        {
+            files: "*.json",
```

Comment:
Why the override? Not complaining, I just don't understand. Is 2 spaces standard for JSON?

Reply:
I was told my @BrandonPacewic that is was. I believe it's because there's so much deep nesting in json, two spaces makes it more readable.

Reply:
All of our `.json` files up to this point have been 2 spaces. Don't think that there is a real "standard" but it's what we have been using.